integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwBigCMMiiDomain.java

```diff
@@ -0,0 +1,378 @@
+        "@@PROP:K8S_NODEPORT_HOST@@:@@PROP:DBPORT@@",
+        String.format("%s:%s", K8S_NODEPORT_HOST, Integer.toString(dbNodePort))),
+        "Can't replace the string @@PROP:K8S_NODEPORT_HOST@@:@@PROP:DBPORT@@ in " + targetModelFile);
+
+    final List<String> modelList = Collections.singletonList(targetModelFile.toString());
+    String jrfMii1Image = createMiiImageAndVerify(
+        "jrf-mii-image",
+        modelList,
+        Collections.singletonList(MII_BASIC_APP_NAME),
+        FMWINFRA_IMAGE_NAME,
+        FMWINFRA_IMAGE_TAG,
+        "JRF",
+        false);
+
+    // push the image to a registry to make it accessible in multi-node cluster
+    dockerLoginAndPushImageToRegistry(jrfMii1Image);
+
+    // create the domain object
+    Domain domain = createDomainResource(domainUid,
+        jrfDomainNamespace,
+        adminSecretName,
+        OCIR_SECRET_NAME,
+        encryptionSecretName,
+        rcuaccessSecretName,
+        opsswalletpassSecretName,
+        replicaCount,
+        jrfMii1Image);
+
+    createDomainAndVerify(domain, jrfDomainNamespace);
+    verifyDomainReady(jrfDomainNamespace, domainUid);
+    // check if multiple configmaps are created
+    try {
+      if (!Kubernetes.listConfigMaps(jrfDomainNamespace).getItems().isEmpty()) {
+        logger.info("Getting Config Maps List");
+        int cmTotalSize = 0;
+        List<V1ConfigMap> items = Kubernetes.listConfigMaps(jrfDomainNamespace).getItems();
+        List<V1ConfigMap> itemsCM = new ArrayList<>();
+        for (var item : items) {
+          if (item.getMetadata().getName().contains("introspect")) {
+            logger.info("Found ConfigMap " + item.getMetadata().getName());
+            logger.info("Found ConfigMap size " + item.toString().getBytes("UTF-8").length + " bytes");
+            cmTotalSize = cmTotalSize + item.toString().getBytes("UTF-8").length;
+            itemsCM.add(item);
+          }
+        }
+        assertTrue((cmTotalSize > 1000000) && (itemsCM.size() > 1),
+            "Produced introspector domain cm is bigger than 1M and was not splitted");
+      }
+    } catch (Exception ex) {
+      throw new RuntimeException("Failed to process config maps" + ex.getMessage());
+    }
+  }
+
+  /**
+   * Construct a domain object with the given parameters that can be used to create a domain resource.
+   * @param domainUid unique Uid of the domain
+   * @param domNamespace  namespace where the domain exists
+   * @param adminSecretName  name of admin secret
+   * @param repoSecretName name of repository secret
+   * @param encryptionSecretName name of encryption secret
+   * @param rcuAccessSecretName name of RCU access secret
+   * @param opssWalletPasswordSecretName name of opss wallet password secret
+   * @param replicaCount count of replicas
+   * @param miiImage name of model in image
+   * @return Domain WebLogic domain
+   */
+  private Domain createDomainResource(
```

Comment:
Move this method to a Utility Class as the same method with same signature is defined in ItFmwMiiDomain.java

Reply:
done